task-random-item.js

```diff
@@ -0,0 +1,8 @@
+//3. Napisz funkcję randomItem(), która zwróci losowy element podanej tablicy.
+
+var array = [1, 2, 3, 4, 5, 6];
```

Comment:
Please create two different arrays and check if the function works for both of them. It's good always check different examples.